src/Illuminate/Contracts/Cache/Repository.php

```diff
@@ -3,8 +3,9 @@
 namespace Illuminate\Contracts\Cache;
 
 use Closure;
+use Psr\SimpleCache\CacheInterface;
 
-interface Repository
+interface Repository extends CacheInterface
```

Comment:
Nevertheless @deleugpn make sure to add this `"psr/simple-cache": "~1.0"` in `Illuminate/Contracts/composer.json` too 😉